modules/contracts/adapters/dex/src/exchanges/exchange_resolver.rs

```diff
@@ -4,20 +4,32 @@ use abstract_adapter_utils::identity::is_current_chain;
 use abstract_dex_adapter_traits::{DexCommand, DexError, Identify};
 use cosmwasm_std::Env;
 
+#[cfg(feature = "juno")]
 use crate::exchanges::junoswap::{JunoSwap, JUNOSWAP};
+#[cfg(feature = "kujira")]
 use crate::exchanges::kujira::{Kujira, KUJIRA};
+#[cfg(feature = "terra")]
 use crate::exchanges::terraswap::{Terraswap, TERRASWAP};
+#[cfg(any(feature = "terra", feature = "neutron"))]
 use abstract_astroport_adapter::{dex::Astroport, ASTROPORT};
+#[cfg(feature = "osmosis")]
 use abstract_osmosis_adapter::{dex::Osmosis, OSMOSIS};
+#[cfg(feature = "juno")]
 use abstract_wyndex_adapter::{dex::WynDex, WYNDEX};
 
 pub(crate) fn identify_exchange(value: &str) -> Result<Box<dyn Identify>, DexError> {
     match value {
+        #[cfg(feature = "juno")]
         JUNOSWAP => Ok(Box::<JunoSwap>::default()),
```

Comment:
Something like that and it should be a little easier to maintain those features
```suggestion
        crate::exchanges::junoswap::JUNOSWAP => Ok(Box::<crate::exchanges::junoswap::JunoSwap>::default()),
```